data/pathfinder/paizo/roleplaying_game/mythic_adventures/ma_equipmods.lst

```diff
@@ -4,18 +4,18 @@
+Mythic Bane		KEY:Special Ability ~ Mythic Bane ~ Weapon	TYPE:Weapon	PLUS:1			VISIBLE:QUALIFY	SOURCEPAGE:p.143	PRETYPE:1,WeaponEnhancement	PRETYPE:1,Weapon		SPROP:A mythic bane weapon works exactly like a bane weapon, except it has a higher bonus and deals extra damage against any creature with the mythic subtype. This ability stacks with the bane weapon special ability.
+Potent		KEY:Special Ability ~ Potent ~ Weapon		TYPE:Weapon	PLUS:2			VISIBLE:QUALIFY	SOURCEPAGE:p.143	PRETYPE:1,WeaponEnhancement	PRETYPE:1,Weapon		SPROP:As a swift action, the wielder can expend one use of mythic power to increase the weapon's enhancement bonus by half his mythic tier (minimum 1, to a maximum of a +6 enhancement bonus) and give it the ability to bypass damage reduction. These benefits last for 1 round.
+Sacrosanct		KEY:Special Ability ~ Sacrosanct ~ Weapon		TYPE:Weapon			COST:5000	VISIBLE:QUALIFY	SOURCEPAGE:p.143	PRETYPE:1,WeaponEnhancement	PRETYPE:1,Weapon		SPROP:Once per day when the wielder uses the sacrosanct weapon to channel energy, she can increase the radius of the channel energy to 40 feet. They wielder must be able to channel positive or negative energy to use this ability. Alternatively, the wielder can expend one use of mythic power to use this ability instead of expending its daily use. When using mythic power to activate this ability, it can be done at will.
 
 ###Block:
 # Modifier Name	Unique Key						Type																							Visible		Type granted		Keys to replace						Source Page		Required Type		Var. Min. Value																Ability
-Legendary		KEY:Legendary Item				TYPE:Armor.Shield.Weapon.Belt.Body.Chest.Eyes.Feet.Hands.Head.Headband.Neck.Ring.Shoulders.Wrist.Bracer.Cape.Instrument.Staff.Rod.Wand				ITYPE:Legendary										SOURCEPAGE:p.169	PRETYPE:1,Enhancement																			ABILITY:Special Ability|AUTOMATIC|Legendary Item ~ Legendary Power|Legendary Item ~ Legendary Surge|Legendary Item ~ Mythic Bond
+Legendary		KEY:Legendary Item				TYPE:Armor.Shield.Weapon.Belt.Body.Chest.Eyes.Feet.Hands.Head.Headband.Neck.Ring.Shoulders.Wrist.Bracer.Cape.Instrument.Staff.Rod.Wand				ITYPE:Legendary										SOURCEPAGE:p.169	PRETYPE:1,ArmorEnhancement,WeaponEnhancement																			ABILITY:Special Ability|AUTOMATIC|Legendary Item ~ Legendary Power|Legendary Item ~ Legendary Surge|Legendary Item ~ Mythic Bond
```

Comment:
Look at the TYPE tag. This can be other types of items than just weapons or armor.